RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -716,6 +733,7 @@ boolean L11_getUVCompass()
 		return false;		//impossible to get compass in this path. [The Shore, Inc] is unavailable
 	}
 
+	pullXWhenHaveY($item[Shore Inc. Ship Trip Scrip], 1, 0);
```

Comment:
do we actually want to spend a pull on scrip? seems too wasteful as it only costs 3 adv and 500 meat